public/coffeescripts/directives.coffee

```diff
@@ -0,0 +1,36 @@
+(->
+  calendarDirective=()->
+    compile=(element,attr)->       
+     
+      cal=document.querySelector(".date")
+
+      pickmeup(cal, {
+        flat : true,
+        mode : 'multiple'
+      })
+
+      cal.addEventListener('pickmeup-change', (e) ->
+        key=$("#selMember option:selected").attr('data-id')
+        member=JSON.parse(localStorage.getItem(key))
+        localStorage.removeItem(key)
+
+        if document.querySelector('#selDay').value=='Відпрацьовані'
```

Comment:
pointing to element value is a bad practice, create another directive for that element and use ng-model